code/tasks/SS3MigrateAssetsFromS3.php

```diff
@@ -16,13 +16,15 @@ public function run($request)
             throw new RuntimeException("This can only be executed from the commandline");
         }
 
-        // The context for the following {src} parameter will be relative to "framework", so "src=../{src}" for example
-        $sourceFolder = trim($request->getVar('src'), "/");
+        // path can be relative or absolute
```

Comment:
Would be good to keep a note here regarding the CWD being framework if someone decides to use a relative path.

Reply:
I've improved the comments a bit more, this is addressed on the class comment block.